govcd/api_vcd_test.go

```diff
@@ -86,6 +86,17 @@ type TestConfig struct {
 		User            string `yaml:"user"`
 		Password        string `yaml:"password"`
 		Token           string `yaml:"token"`
+		UseSamlAdfs     bool   `yaml:"useSamlAdfs"`
+		CustomAdfsRptId string `yaml:"customAdfsRptId"`
+
+		// The below `SamlUser`, `SamlPassword` and `SamlCustomRptId` variables are optional and are
+		// related to additional test run specifically with SAML user/password. It can be useful in
```

Comment:
Are we talking about a single test ("to an additional test") or multiple ones ("to additional tests") ?

Also, the meaning of "It" in the second paragraph is ambiguous.

Reply:
At the moment it is single test. One never knows when more appear. changed.